std/functional.d

```diff
@@ -1327,27 +1327,19 @@ unittest
         static int func_trusted() @trusted { return 5; }
         static int func_system() @system { return 6; }
         static int func_pure_nothrow() pure nothrow { return 7; }
-        static int func_pure_nothrow_safe() pure @safe { return 8; }
+        static int func_pure_nothrow_safe() pure nothrow @safe { return 8; }
```

Comment:
Looks like this is a bug below, the test for it is commented out. Should it be restored? `dg_safe` test is commented out below as well.